src/components/admin/form-components/validated-fields.jsx

```diff
@@ -0,0 +1,15 @@
+import TextField from 'material-ui/TextField';
+
+// Higher Order Components
+import withValidate from 'components/admin/form-components/withValidate';
+
+export const hasHttps = (value) => {
+  if (value && value.length > 4 && value.substr(0, 5) !== 'https') {
+    return (
+      'You are saving an image without using https.'
+    );
+  }
+  return null;
+};
+
+export const ValidatedImage = withValidate(hasHttps)(TextField);
```

Comment:
I would maybe be a bit more verbose in the component name and call it something like `ValidatedImageReference` to stress that we are not uploading anything.

Reply:
Yeah I agree, or maybe just `ValidatedImageUrlField`, I don't mind it being a bit long if it's clear

Reply:
Agree - TODO